Automattic/pocket-casts-android
# Transcripts - Tracks

## Description
This PR adds Track events for transcripts.

## Testing Instructions
** `player_shelf_action_tapped` event is tested in https://github.com/Automattic/pocket-casts-android/pull/2512
** Below events are tracked with podcast and episode uuids as properties

1. Play an episode having transcript
2. Tap on the player shelf transcript button
3. Notice that `transcript_shown` is tracked 
4. Pull to refresh
5. Notice that `transcript_pulled_to_refresh` is tracked 
6. Go back either by tapping close button or by swipe back gesture
7. Notice that `transcript_dismissed` is tracked 
8. Play another episode with a transcript that you haven't opened before
9. Disable internet
10. Tap transcript shelf icon on the player
11. Notice that an error message is shown and `transcript_error` is tracked
12. Enable internet and tap "Try again"
13. Notice that transcript is shown and `transcript_shown` is tracked 
14. Tap search button
15. Notice `transcript_search` is tracked
16. Search for a term and tap next button in the search bar
17.  Notice `transcript_search_next_result` is tracked
18. Tap previous button in the search bar
19. Notice that `transcript_search_previous_result` is tracked

## Checklist

#### I have tested any UI changes...
